### PR TITLE
[Binding addons starting with "S"] Correct linter errors only

### PR DESCRIPTION
**Pull request by bgilmer77**
Signed-off-by: Brad Gilmer <brad@gilmer.tv>

### CODE REVIEW

File: addons/binding/org.openhab.binding.sensebox/README.md [hunk 1]
```diff
@@ -37,27 +37,25 @@ Thing sensebox:box:home [ senseBoxId = "foothesmurfingbar", refreshInterval = 60
...
-| measurements#particulateMatter10                | Temperature Matter 10 µm in diameter                         | Number              | False    |
-| lastReported#uvIntensityLastReported            | The timestamp when uv radiation intensity was last reported  | DateTime            | True     |
-| lastReported#luminanceLastReported              | The timestamp when illuminance was last reported             | DateTime            | True     |
-| lastReported#pressureLastReported               | The timestamp when pressure was last reported                | DateTime            | True     |
-| lastReported#humidityLastReported               | The timestamp when humidity was last reported                | DateTime            | True     |
-| lastReported#temperatureLastReported            | The timestamp when temperature was last reported             | DateTime            | True     |
-| lastReported#particulateMatter2dot5LastReported | The timestamp when particulate matter 2.5 was last reported  | DateTime            | True     |
-| lastReported#particulateMatter10LastReported    | The timestamp when particulate matter 10 was last reported   | DateTime            | True     |
+
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.sensebox/README.md]: Uhm....

**Comment by bgilmer77** [comment]
er...

File: addons/binding/org.openhab.binding.sensebox/README.md [hunk 2]
```diff
@@ -37,27 +37,42 @@ Thing sensebox:box:home [ senseBoxId = "foothesmurfingbar", refreshInterval = 60
...
+To see what it's all about, try one of these examples, or format your own.
+
+Load: Example 1 - Example 2 - Example 3
+
+For more information:
+I'm on Twitter as @TheIdOfAlan
+I sometimes post on my personal site alanwsmith.com
+This is an Open Source GitHub Project.
+It has a Jasmine Test Suite.
+What to show your appreciation? Buy me a book
```
Comment (ThomDietrich) [inline addons/binding/org.openhab.binding.sensebox/README.md]: @bgilmer77 double uhm? :)

**Comment by bgilmer77** [comment]
Triple-checked.  Ok to merge now?

**Pull request merged**